### ISSUE TITLE
minLevel does not respected

### ISSUE THREAD

**Issue by anatoli-klamer** [rapid7/r7insight_node#29]
When I set minLevel to 'info' , r7insight_node ignore minLevel and still publish debug logs

![image](https://user-images.githubusercontent.com/87621606/127111327-58e6198b-5cca-4aad-8931-19719652b533.png)

```
new winston.transports.Insight({
		token,
		region: 'eu',
		level: 'info',
		json: true, //<< -----
	})```

**Comment by sbialkowski-r7** [comment]
Hi @anatoli-klamer!

I can't seem to reproduce this on my end with this UT:
```javascript
tape('String logger minLevel option is supported and works', function (t) {
  t.plan(1);
  t.timeoutAfter(1000);
  const logger = new Logger({
    token,
    region: 'eu',
    minLevel: 'info',
  });

  mockTest(function (data) {
    t.fail('Data should not be sent with lower logger level.');
  });

  logger.debug('asd');
  logger.log('debug', 'asd');
  logger['debug']('asd');

  t.pass('Test finished');
});
```

could you perhaps give me a snippet of your code where this is happening?

based on your screeenshot, I believe you're using the library incorrectly unless I'm mistaken
you're doing something like
```javascript
logger.log({level: 'debug', some_message_here: 'hello});
```

and as far as I know, we don't read the logger message in order to figure out the logging level, you need to one of the following:
```javascript
  logger.debug({some_message_here: 'hello'});
  logger.log('debug', {some_message_here: 'hello'});
```

**Comment by anatoli-klamer** [comment]
i think the reason is we are using json logs:
try this:
```
new winston.transports.Insight({
		token,
		region: 'eu',
		level: 'info',
		json: true, //<< -----
	})

```

**Comment by sbialkowski-r7** [comment]
Hey @anatoli-klamer

this should be fixed with the published `3.2.0` release, hope it works perfectly, feel free to open another issue if you've any more problems!
https://www.npmjs.com/package/r7insight_node/v/3.2.0
